apps/short/backend.production.yaml

```diff
@@ -138,6 +138,14 @@ spec:
                 secretKeyRef:
                   name: short
                   key: google-api-key
+            - name: ENABLE_ENCRYPTION
+              value: "true"
+            - name: CERT_FILE_PATH
+              value: /etc/certs/tls.crt
+            - name: KEY_FILE_PATH
+              value: /etc/certs/tls.key
+            - name: GRPC_API_PORT
+              value: "8081"
```

Comment:
Did you remember to attach the file to the container? https://github.com/short-d/gitops/blob/master/apps/kgs/production.yaml#L88-L96

Reply:
And here: https://github.com/short-d/gitops/blob/master/apps/kgs/production.yaml#L30-L33